static-site/src/components/splash/index.jsx

```diff
@@ -211,61 +213,179 @@ export default Splash;
 /*** SHOWCASE ***/
 
 const UrlShowcaseTile = ({ card }) => {
+
+  /* Narrative detection works for all three sources:
+  1. Core: /narratives/<narrative path>
+  2. Community: /community/narratives/<owner>/<narrative path>
+  3. Groups: /groups/<group>/narratives/<narrative path>
+
+  Including slashes in the check prevents false positives.
+  */
+  const isNarrative = card.url.includes('/narratives/');
+
   return (
     <CardOuter>
       <CardInner>
         <a href={card.url}>
-          <CardTitle $squashed>
+          <CardTitle>
             {card.name}
           </CardTitle>
-          <CardImgWrapper filename={card.img}/>
+          <CardImgContainer>
+            <CardImgWrapper filename={card.img}/>
+            <InfoIcons>
+              <CardSourceIcon url={card.url} isNarrative />
+              {isNarrative && <NarrativeIcon />}
+            </InfoIcons>
+          </CardImgContainer>
         </a>
+        <CardDescription>
+          {card.description}
+        </CardDescription>
       </CardInner>
     </CardOuter>
   )
 }
 
-const cardWidthHeight = 160; // pixels
+const cardWidth = 220; // pixels
+const cardHeight = 285; // pixels
+
+function CardSourceIcon({ url, isNarrative }) {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const gitHubLogo = require(`../../../static/logos/github-mark.png`).default.src;
+
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const nextstrainLogo = require(`../../../static/logos/nextstrain-logo-tiny.png`).default.src;
+
+  let maintainers, image;
+
+  if (url.startsWith('/community')) {
```

Comment:
Implicit in this logic is that the urls (as defined in the YAML) will always have a leading `/`. But the links continue to work if we forget this, and we never assert it anywhere. As far as I know we can't test for this at build time, due to the async nature of `import`s (although I feel like we _should_ be able to somehow 🤷 ) but it'd be easy to add a (jest) test for this.

All of the URLs currently do have leading slashes, so this can be split out into another issue.

Reply:
This came back to bite in https://github.com/nextstrain/nextstrain.org/pull/993#discussion_r1729422975. Removing the note and properly handling the absence of `/` in #996.